libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -522,6 +528,90 @@ class UdpContext
         reinterpret_cast<UdpContext*>(arg)->_recv(upcb, p, srcaddr, srcport);
     }
 
+#if LWIP_VERSION_MAJOR == 1
+    /*
+     * Code in this conditional block is copied/backported verbatim from
+     * LwIP 2.1.2 to provide pbuf_get_contiguous.
+     */
+
+    static const struct pbuf *
+    pbuf_skip_const(const struct pbuf *in, u16_t in_offset, u16_t *out_offset)
+    {
+      u16_t offset_left = in_offset;
+      const struct pbuf *q = in;
```

Comment:
what is q? please either add a comment, or rename to something meaningful (e.g. it which is typical of iterator, or similar)

Reply:
This code was copy-pasted from LwIP2 source, nevertheless I renamed it to`pbuf_it`.